### PR TITLE
Change promotion page

### PR DESCRIPTION
**Pull request by cheng203**
Still have problem with layout stuff. Will discuss later in the meeting

### CODE REVIEW

File: src/pages/Promotion/Content.js [hunk 1]
```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
...
                     </Col>
-                    <Col span="6"></Col>
+                    <Col span="10"></Col>
                 </Row>
-                <Row>
-                    <Col className="page-transient"/>
+                <Row className="part2">
+                    <Col xs={1} sm={1} md={2} lg={2} xl={3} xxl={3}/>
+                    <Col xs={22} sm={22} md={20} lg={20} xl={18} xxl={18}>
+                        <img src="/assets/images/carousel.png" alt="carousel"></img>
```
Comment (dudongtai2002) [inline src/pages/Promotion/Content.js:40]: 1. The left image seems changed, could we use the latest one?
2. Is there any way we can improve the quality of this image? It seems quite blur on my browser.

Comment (cheng203) [reply src/pages/Promotion/Content.js:40]: needs to be changed in the figma. Will tell minz do so

File: src/pages/Promotion/Content.js [hunk 2]
```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
...
                     </Col>
-                    <Col span="2"></Col>
+                    <Col xs={4} sm={4} md={3} lg={4} xl={5} xxl={6}></Col>
+                </Row>
+                <Row className="part8" gutter={83}>
+                    <Col xs={1} sm={1} md={1} lg={3} xl={3} xxl={5}></Col>
+                    <Col xs={11} sm={11} md={11} lg={9} xl={9} xxl={7}>
+                        <Card bordered={false}>
+                            <div className="part8-text-title">Never Miss One Health Service</div>
+                            <div className="part8-text-content">With a calendar to integrate. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
```
Comment (dudongtai2002) [inline src/pages/Promotion/Content.js:96]: why Latin here - -

File: src/pages/Promotion/Content.js [hunk 3]
```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
...
+                    <Col xs={1} sm={1} md={1} lg={3} xl={3} xxl={5}></Col>
+                    <Col xs={11} sm={11} md={11} lg={9} xl={9} xxl={7}>
+                        <Card bordered={false}>
+                            <img src="/assets/images/wellness.png"/>
+                        </Card>
+                    </Col>
+                    <Col xs={11} sm={11} md={11} lg={9} xl={9} xxl={7}>
+                        <Card bordered={false}>
+                            <div className="part9-text-title">Intagrate Your Health Data in One Platform</div>
+                            <div className="part9-text-content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
```
Comment (dudongtai2002) [inline src/pages/Promotion/Content.js:116]: Again, why Latin here - -

File: src/pages/Promotion/Content.js [hunk 4]
```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
...
+                        </Card>
+                    </Col>
+                    <Col xs={1} sm={1} md={1} lg={3} xl={3} xxl={5}></Col>
+                </Row>
+                <Row className="part10" gutter={83}>
+                    <Col xs={1} sm={1} md={1} lg={3} xl={3} xxl={5}></Col>
+                    <Col xs={11} sm={11} md={11} lg={9} xl={9} xxl={7}>
+                        <Card bordered={false}>
+                            <div className="part10-text-title">Family-shared Health Management</div>
+                            <div className="part10-text-content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
```
Comment (dudongtai2002) [inline src/pages/Promotion/Content.js:126]: same here

File: src/pages/Promotion/Content.js [hunk 5]
```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
...
                 </Row>
                 <Row className="part6">
-
+                    <div className="part6-header">- Future -</div>
+                    <div className="part6-title">Unlock New Features Overtime</div>
+                    <div className="part6-subtitle">Your continous-use of Dr.Chain can also unlock new features.</div>
+                    <div className="part6-background">
+                        <Col xs={0} sm={0} md={2} lg={4} xl={5} xxl={5}></Col>
+                        <Col xs={24} sm={24} md={17} lg={15} xl={14} xxl={14}>
+                            <img src="/assets/images/promotion_bot.png" alt="heart"/>
```
Comment (dudongtai2002) [inline src/pages/Promotion/Content.js:204]: seems this png is not responsive with the zoom in/out of the browser, is there any possibility we could explore to fix this?

**Review by dudongtai2002 — COMMENTED** [review]
Great work overall! There are some minor comments that could be resolved. 

One additional issue, the bottom of the promotion page seems are missing the 'Dr. Chain' on the left, and might need some minor modification of the alignment

**Review by regulus17 — APPROVED** [review]

**Pull request merged**